<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Client</string>
    </property>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>97</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHost">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.44</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPort">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>9879</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelConnected">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;rgb(0, 152, 111)
;</string>
     </property>
     <property name="text">
      <string>Listening</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSocketInfo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>461</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
